buku

```diff
@@ -614,7 +614,7 @@ class BukuDb:
             conn.commit()
         except Exception as e:
             LOGERR('initdb(): %s', e)
-            sys.exit(1)
+            raise e
```

Comment:
`sys.exit()` is only appropriate for native CLI invocations (Buku should not kill an application using it as a library :sweat_smile:)